package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dyson",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
Please don't update the `version`, I do releases with [release-it](https://github.com/release-it/release-it).

Reply:
Got it.